internal/databases/sqlserver/log_restore_handler.go

```diff
@@ -68,13 +86,21 @@ func HandleLogRestore(backupName string, untilTS string, dbnames []string, fromn
 					logBackupName, fromname)
 				continue
 			}
-			isLastOne, err := restoreSingleLog(ctx, db, folder, logBackupName, dbname, fromname, stopAt)
+			isLastOne, CurrentBackupFinishdate, err := restoreSingleLog(ctx,
```

Comment:
`isLastOne` return argument is no longer needed as it computed using `backupFinishDate`.
We can return just `backupFinishDate`